core/include/spi.h

```diff
@@ -56,5 +56,7 @@ struct spi_ops {
 		size_t *num_rxpkts);
 };
 
+void spi_init(void);
```

Comment:
I mirrored this after `void console_init(void);` in console.h. Also, as mentioned above (https://github.com/OP-TEE/optee_os/pull/1161#discussion_r86809938), this allows for static TAs to `#include "spi.h"` and call into the spi init/test code. If not, where should I put it?